02-express-tutorial/app.js

```diff
@@ -1,11 +1,42 @@
 const express = require("express");
-const { products } = require("./data");
+const { products, people } = require("./data");
+const peopleRouter = require('./routes/people');
 const app = express();
 
+function logger(req, res, next){
+  const currentTime = new Date();
+  console.log(`${currentTime} ${req.method} ${req.url}`);
+  next();
+}
+
 app.listen(3000, () => {
   console.log("Server is running on port 3000");
 });
 
+app.use(logger);
+
+//app.get("/api/v1/people/", (req, res) => {
+//  res.json(people);
+//});
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
+
+app.use("/api/v1/people", peopleRouter);
+
+// app.post("/api/v1/people/", (req, res) => {
```

Comment:
for blocks like this JS offers multi-line comment syntax - wrapping the entire block in `/* ... */` More here - https://www.w3schools.com/js/js_comments.asp